### PR TITLE
feat: new table ui design

### PR DESCRIPTION
**Pull request by 0tuedon**
1. Removed the tags and categories column, merged the title column and tag column together
Screenshot 2023-08-01 at 01 06 25

2.  Renamed the tiltle column and added the Sats column

3. how the sats are calculated <2500 = low, < 5000 == medium <7500 === high , > 10000===  very-high (Also there are .pngs related to this calculation

@adamjonas  @Extheoisah  @Emmanuel-Develops

### CODE REVIEW

**Comment by BalogunofAfrica** [comment]
@0tuedon I think you need to rebase on `dev`, there are commits that shouldn't be showing here

**Comment by 0tuedon** [comment]
> @0tuedon I think you need to rebase on `dev`, there are commits that shouldn't be showing here

Yh would rebase. 
Just saw that too.

**Comment by 0tuedon** [comment]
> > @0tuedon I think you need to rebase on `dev`, there are commits that shouldn't be showing here
> 
> Yh would rebase. Just saw that too.

@BalogunofAfrica  i have rebased and pushed

**Comment by adamjonas** [comment]
Can you add the categories as well to this? Only the tags are showing up.

**Comment by 0tuedon** [comment]
> Can you add the categories as well to this? Only the tags are showing up.

Screenshot 2023-08-01 at 22 24 50
@adamjonas  added the categories ,

2. I also filtered the None tag which came as a string on the transcripts @Extheoisah  @Emmanuel-Develops

File: src/components/tables/CurrentJobsTable.tsx [hunk 1]
```diff
@@ -81,35 +87,46 @@ const CurrentJobsTable = () => {
     () =>
       [
         {
-          name: "title",
-          type: "text-long",
-          modifier: (data) => data.content.title,
+          name: "Talk Title",
```
Comment (adamjonas) [inline src/components/tables/CurrentJobsTable.tsx:90]:
```suggestion
          name: "Title",
```
Let's restore to title please

File: src/components/tables/PastJobsTable.tsx [hunk 2]
```diff
@@ -1,34 +1,37 @@
...
 import { GroupedLinks, ReviewStatus } from "./TableItems";
+import TitleWithTags from "./TitleWithTags";
 import type { TableStructure } from "./types";
 
 const tableStructure = [
   {
-    name: "title",
-    type: "text-long",
-    modifier: (data) => data.content.title,
+    name: "Talk Title",
```
Comment (adamjonas) [inline src/components/tables/PastJobsTable.tsx:14]:
```suggestion
    name: "Title",
```

File: src/components/tables/QueueTable.tsx [hunk 3]
```diff
@@ -190,46 +197,63 @@ const QueueTable = () => {
     () =>
       [
         {
-          name: "title",
-          type: "text-long",
-          modifier: (data) => data.content.title,
+          name: "Talk Title",
```
Comment (adamjonas) [inline src/components/tables/QueueTable.tsx:200]:
```suggestion
          name: "Title",
```

File: src/utils.ts [hunk 4]
```diff
@@ -237,3 +237,44 @@ export function isReviewPending(review: UserReview) {
...
+export const tagColors = ["#9C007A", "#16863C", "#9C3800", "#00519C"];
+
+export const transcriptsCategories = [
+  {
+    name: "🎙️ Conference",
+    slug: "conference",
+  },
+  { name: "💻 Core-Dev-Tech", slug: "core-dev-tech" },
+  { name: "💻 Hackathon", slug: "hackathon" },
+  { name: " Meeting", slug: "meeting" },
```
Comment (adamjonas) [inline src/utils.ts:266]: let's figure out an emoji for this

**Review by adamjonas — COMMENTED** [review]
This is close!

The main thing is that the text font for the labels is too big.

font-size: 11px or 12px is about right.
Screenshot 2023-08-01 at 7 18 31 PM

**Comment by 0tuedon** [comment]
okay, reducing text size yh?

File: src/components/tables/CurrentJobsTable.tsx (hunk 1 above)
Comment (0tuedon) [reply src/components/tables/CurrentJobsTable.tsx:90]: Alright.

**Comment by 0tuedon** [comment]
> This is close!
> 
> The main thing is that the text font for the labels is too big.
> 
> font-size: 11px or 12px is about right. Screenshot 2023-08-01 at 7 18 31 PM

These new commits achieve this.

File: src/components/tables/TitleWithTags.tsx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+                textTransform="capitalize"
+                color="#FCFCFC"
+              >
+                {foundCategories?.name}
+              </Text>
+            </Box>
+          )}
+          {allTags
+            .slice(0, 2)
+            .filter((tags) => tags.toLocaleLowerCase() !== "none")
```
Comment (Extheoisah) [inline src/components/tables/TitleWithTags.tsx:53]: why use `toLocaleLowerCase`? pls use `toLowerCase`

File: src/utils.ts [hunk 6]
```diff
@@ -237,3 +237,44 @@ export function isReviewPending(review: UserReviewData) {
     review.archivedAt === null
   );
 }
+
+// Convert a string to an array
+export const convertStringToArray = (text: string[] | string) => {
+  let stringArray = text as string;
+  if (stringArray[0] === "[") {
+    // eslint-disable-next-line prettier/prettier
```
Comment (Extheoisah) [inline src/utils.ts:245]: why do we have this line? if its a prettier problem, we should fix it.

File: src/utils.ts [hunk 7]
```diff
@@ -237,3 +237,44 @@ export function isReviewPending(review: UserReviewData) {
...
+    let _parsed = stringArray
```
Comment (Extheoisah) [inline src/utils.ts:246]: no need to prefix the variable with an underscore. Also, if we ain't re-assigning the variable, lets make it a `const`

**Review by Extheoisah — COMMENTED** [review]
a few comments

**Pull request merged**